cmd/query/app/server.go

```diff
@@ -346,10 +347,7 @@ func (s *Server) GRPCAddr() string {
 
 // Close stops HTTP, GRPC servers and closes the port listener.
 func (s *Server) Close() error {
-	errs := []error{
-		s.queryOptions.TLSGRPC.Close(),
-		s.queryOptions.TLSHTTP.Close(),
-	}
+	errs := []error{}
```

Comment:
nit: no need to allocate slice (doesn't matter much for Close function, but as a good practice for hot code)
```suggestion
	var errs []error
```